test/manual-smock/MockBCoWHelper.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import {
+  BCoWHelper,
+  BMath,
+  GPv2Interaction,
+  GPv2Order,
+  GetTradeableOrder,
+  IBCoWFactory,
+  IBCoWPool,
+  ICOWAMMPoolHelper,
+  IERC20
+} from '../../src/contracts/BCoWHelper.sol';
+import {Test} from 'forge-std/Test.sol';
+
+contract MockBCoWHelper is BCoWHelper, Test {
+  // NOTE: manually added methods (internal immutable exposers not supported in smock)
+  function call__APP_DATA() external view returns (bytes32) {
```

Comment:
why not make `_APP_DATA` public?

Reply:
is not required in the interface that CoWSwap provides `ICOWAMMPoolHelper`, they only request to expose `factory`, `tokens` and `order`